src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1696,8 +1696,6 @@ namespace ReportSizingManager {
 							}
 							SupFanNum = PrimaryAirSystem( CurSysNum ).SupFanNum;
```

Comment:
SimAirServingZones::InitAirLoops sets supFanModelTypeEnum, and SupFanNum or supFanVecIndex when the supply fan is directly on the main branch [here](https://github.com/NREL/EnergyPlus/blob/173bb58bcc95ec005c91946ecee8f573224904fd/src/EnergyPlus/SimAirServingZones.cc#L1866-L1939).  (similar for return fan)

If the fan is a child of a unitary system, it is ignored here.  Only the unitary system tries to set this [here] in HVACUnitarySystem::SizeUnitarySystem, but it needs to be sensitive to the fan object type.  I can add that here, but there's still a question then about other unitarysystems.

The I/O Ref (v8.7 p. 645) says "Central supply and return fan temperature rise is taken into account in sizing the central cooling coils."  But it appears that is true only when the fan is directly on the branch or if the fan is inside an AirloopHVAC:UnitarySystem.  Do we need to add a similar block of code to other unitarysystem types?  And I wonder if the code in InitAirLoops gets the right fan for dual-fan DDVAV systems?  I guess this part should be a new issue.  Fix the unitary system here and leave the rest.